app/frontend/src/components/Welcome.vue

```diff
@@ -9,13 +17,13 @@
         <ul>
           <li class="my-2">
             <strong>New User:</strong> Go to
-            <router-link :to="{ name: 'RequestAccount' }">"Request Account"</router-link> and submit the form to add your application.
+            <router-link to="">"Request Account"</router-link> and submit the form to add your application.
             <br />* Check your email for the confirmation, then you’re ready to go!
           </li>
           <li class="my-2">
             <strong>Existing User:</strong> Manage your application from
             <router-link :to="{ name: 'MyApps' }">"My Applications"</router-link> or
-            <router-link :to="{ name: 'RequestAccount' }">"Request Account"</router-link> for a new application
+            <router-link to="">"Request Account"</router-link> for a new application
```

Comment:
These should outright no longer be hyperlinks. I'd recommend making it just text without the router-link attribute completely.